packages/protocol/test/common/multisig.ts

```diff
@@ -35,21 +79,26 @@ contract('MultiSig', (accounts: any) => {
   describe('#submitTransaction()', () => {
     let txData: string
     beforeEach(async () => {
-      // @ts-ignore contract is a property
-      txData = multiSig.contract.addOwner.getData(accounts[2])
+      txData = encodeAddOwnerFunc(web3, accounts[2])
```

Comment:
The `multiSig` object already has the ABI stored, why not leverage that via `getData`?

If this was due to https://github.com/celo-org/celo-monorepo/pull/1982/commits/39e33b794ceea96796f91ff6b39f58f38a144460#r353409066, that may have been a miscommunication, since I was suggesting that we shouldn't use `getData` to make a contract call, and not that it wasn't a good way of encoding tx data

Reply:
well, since Truffle V5 getData does has been repaced by the underlying web3 object, hence that would not work anymore. I changed it appropriately